e2e/accounts/accounts_rpc_test.go

```diff
@@ -43,10 +42,7 @@ func (s *AccountsTestSuite) TestRPCEthAccountsWithUpstream() {
 	// FIXME(tiabc): Stop skipping after https://github.com/status-im/status-go/issues/424
 	s.T().Skip()
 
-	s.StartTestBackend(
-		params.RopstenNetworkID,
-		e2e.WithUpstream("https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4"),
-	)
+	s.StartTestBackend(e2e.WithUpstream("https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4"))
```

Comment:
By the way, upstream URLs should be compliant to the network. Otherwise they won't work.

Reply:
Odd. From my time here, this has being the format we had used. Am not sure why this is being used here from the one in `params/default.go` but since it did not related with this PR's change, am not sure if this should be touched. Please your thoughts on this?

Reply:
It wasn't used so far because we'd always run this test on Ropsten. Now that we need to run it on an arbitrary network, we need to specify a proper upstream for that network. Does it make sense?

Reply:
Sure. Now the question is the URL to be specified will this be coming from `params/default.go` or should also be provided as a cli flag?

Reply:
@influx6 this is you who is doing this. What'd you suggest?

Reply:
Well, since we have do have `params/default`, then we will use those has fallbacks, if none is provided through a flag.

Reply:
Do we at all have such a use case when we need to provide a flag with an upstream URL?

Reply:
Well for example, if we had a private statuschain where we need to provide a unix address for such. Of course, if this is not necessary, we can simply stay with the three we currently have.

Reply:
I don't think running tests with upstream for statuschain is worth that hardening. We'll run public testnet tests with upstream in the end anyway. So I suggest we don't do that flag.

Reply:
Ok. Sounds good. Will remove the flag.